vincechain/chain.json

```diff
@@ -0,0 +1,98 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "vincechain",
+  "status": "live",
+  "network_type": "mainnet",
+  "pretty_name": "VinceChain",
+  "chain_id": "vince_1000-1",
+  "bech32_prefix": "vce",
+  "node_home": "$HOME/.vinced",
+  "slip44": 60,
+  "fees": {
+    "fee_tokens": [
+      {
+        "denom": "avce",
```

Comment:
this denom must refer to the minimal denom unit, which you've changed to just 'vce'